src/overlay_ordering.cpp

```diff
@@ -0,0 +1,25 @@
+#include "overlay_ordering.h"
+
+std::map<std::string, int> base_mutation_overlay_ordering;
+std::map<std::string, int> tileset_mutation_overlay_ordering;
+
+void load_overlay_ordering( JsonObject &jsobj )
```

Comment:
Huh? That looks like the very same code as in `cata_tiles::load_tilejson_from_file`. The common code should be in a separate function, probably a function that is given the JsonObject and map where it should store the loaded data (`load_foo(JsonObject &, std::ma<...> &ordering)`). The function can be called from here (with `base_mutation_overlay_ordering` and from cata_tiles (with `tileset_mutation_overlay_ordering`).

Btw. I don't see any code that clears base_mutation_overlay_ordering.